client/src/components/CustomEvents/CustomEventDialog.js

```diff
@@ -164,10 +164,21 @@ class CustomEventDialog extends PureComponent {
                         <Button onClick={() => this.handleClose(true)} color="primary">
                             Cancel
                         </Button>
-
-                        <Button onClick={() => this.handleClose(false)} variant="contained" color="primary">
-                            {this.props.customEvent ? 'Save Changes' : 'Add Event'}
-                        </Button>
+                        <Tooltip
+                            title="Schedule and day must be checked"
+                            disableHoverListener={this.state.scheduleIndices.length && this.state.days.some(Boolean)}
```

Comment:
Was wondering if this statement should be made into a variable, since I use it again in Line 176? And if so, where would that be defined?

Reply:
Good hutch! It's always good to try and write [DRY code](https://en.wikipedia.org/wiki/Don't_repeat_yourself) :)

My suggestion would be to add a function to the component and then call that function in the render. Something like:
```
isDisabled = () => {
    return this.state.scheduleIndices.length && this.state.days.some(Boolean)
}
```
```suggestion
                            disableHoverListener={this.isDisabled()}
```